src/layouts/Collaborateur/updateCollaborator.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/prop-types */
+import Modal from "@mui/material/Modal";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+import Typography from "@mui/material/Typography";
+import { Button } from "@mui/material";
+import { useUpdateCollaboratorMutation } from "services/collaborator/collaborator.api.slice";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { useEffect } from "react";
+
+const formStyle = {
+  display: "flex",
+  flexDirection: "column",
+  gap: "1rem",
+};
+
+export default function UpdateCollaboratorForm(props) {
```

Comment:
```suggestion
export default function UpdateCollaboratorForm({
    collaborator,
    isOpen,
    handleClose,
    setCollaboratorToUpdate,
    existingCollaborators,
    setCollaborateurs,
    style,
  }) {
```

tu peut faire destructuring des props directement ici